src/views/GifReferenceWidget.vue

```diff
@@ -104,6 +139,18 @@ export default {
 		justify-content: center;
 		position: relative;
 
+		.toggle-gifs-button {
+			position: absolute;
+			top: 0;
+			right: 0;
+			padding: 0;
+			border-radius: 50%;
+		}
+
+		.gifs-disabled {
+			margin: 12px !important;
+		}
```

Comment:
```suggestion
		.gifs-disabled {
			margin: 12px !important;
			white-space: initial;
		}
```
To avoid the `<p>` being larger than the text and then the text not being aligned in the widget.
Before:
![image](https://github.com/nextcloud/integration_giphy/assets/11291457/270921e1-dc70-4cb3-b9dc-e9ca96a0cad2)
After:
![image](https://github.com/nextcloud/integration_giphy/assets/11291457/0fecba70-aeac-4250-b71d-90f74259447a)

Reply:
Nice. It not only solves it for you, it stays the same for me!